app/src/LatestMood/index.js

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
```

Comment:
I think you probably want to delete this `LatestMood` component and any references to it now that you are not using it anymore.

Reply:
Good point, I will do that. Thank you!